charts/whatsapp-proxy-chart/values.yaml

```diff
@@ -102,3 +102,8 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+# Enables service monitor for the prometheus operator.
+enableServiceMonitor: false
+prometheus:
+  release: my-prometheus-release
```

Comment:
Will these be updated in runtime?

Reply:
yes, the intention is that the user provides their own values.yaml